apps/web/src/pages/quick-start/QuickStartPage.tsx

```diff
@@ -0,0 +1,59 @@
+import { Center } from '@mantine/core';
+import { useNavigate } from 'react-router-dom';
+import PageMeta from '../../components/layout/components/PageMeta';
+import PageContainer from '../../components/layout/components/PageContainer';
+import { colors, Text, Title } from '../../design-system';
+import React from 'react';
+import { useMutation, useQueryClient } from 'react-query';
+import { updateUserOnBoarding } from '../../api/user';
+import { IUserEntity } from '@novu/shared';
+import { OnboardingSteps } from './components/OnboardingSteps';
+
+function QuickStart() {
+  const queryClient = useQueryClient();
+  const navigate = useNavigate();
+
+  const { mutateAsync: updateOnBoardingStatus } = useMutation<
+    IUserEntity,
+    { error: string; message: string; statusCode: number },
+    { showOnBoarding: boolean }
+  >(({ showOnBoarding }) => updateUserOnBoarding(showOnBoarding));
+
+  async function disableOnboarding() {
+    await updateOnBoardingStatus({ showOnBoarding: false });
+  }
+
+  async function onDismissOnboarding() {
+    await disableOnboarding();
+    await queryClient.refetchQueries('/v1/users/me');
```

Comment:
Do we have query key for this or can we create on and use that instead?

Reply:
I'll add it as a task on linear- if we add that I'd like to do that all around the project, as we use that in a lot of places.